concepts/0003-protocols/semver.md

```diff
@@ -1,93 +1,51 @@
-1.0 features using a 1.2 message, though of course it will lose any features
-that 1.2 added. Thus, accepting such a message could have two possible outcomes:
-
-1. The message at version 1.2 might look and behave exactly like it did at version
-1.0, in which case the message will process without any trouble.
+Within a given major version of a protocol, an agent should:
 
-2. The message might contain some fields that are unrecognized and need to be ignored.
+- specify a minimum minor version supported, defaulting to "0"
+- specify a current minor version that is fully supported and used by default
 
-In case 2, it is best practice for the recipient to send a `problem-report` that
-is a *warning*, not an *error*, announcing that some fields could not be processed
-(code = `fields-ignored-due-to-version-mismatch`). Such a message is *in addition
-to* any response that the protocol demands of the recipient.
+This leads to the following received message handling rules:
 
-If the recipient of a protocol's initial message generates a response, the response
-should use the latest major.minor protocol version that both parties support and
-know about. Generally, all messages after the first use only major.minor
+- message types received with a minor versions below the minimum may be answered with a `report-problem` with code `version-not-supported`
+- message types received with a minor version at or higher than the minimum supported and less than the current minor version are processed, ideally with a response using the same minor version of the received message
+  - The recipient may want to send a warning `report-problem` with code `version-with-degraded-features`
+- message types received with a minor version higher than the current minor version are processed with any unrecognized fields ignored
+  - The recipient may want to send a warning `report-problem` with code `fields-ignored-due-to-version-mismatch` 
 
-[![version negotiation matrix](version-negotiation-matrix.png)](
-https://docs.google.com/spreadsheets/d/1W5KYOqCCqmTeU4Z7XZQH9_6_0TeP5Vf5TtsOZmioyB0/edit#gid=0)
+As documented in the semvar documentation, these requirements may not apply when
+major version 0 is used. Specific rules associated each v0 protocol are left to
+the discretion of the community.
 
+Agents may support multiple major versions and select which major version to
+use when initiating an instance of the protocol.
 
+The [discover features](../../features/0031-discover-features/README.md) protocol
+can be used by one agent to query the versions of protocols supported by another agent,
+informing the selection of which version of a protocol to use in initiating an
+instance of a protocol. Agent receiving a discover features request may respond with a list of
+preferred protocols they support, as described in that protocol.
 
- 
+An agent should reject messages from protocols or unsupported protocol major versions with
```

Comment:
These 3 lines feel pretty redundant with the lines just before them.

Reply:
Removed the previous lines.